cmd/collector/app/builder/span_handler_builder.go

```diff
@@ -45,18 +45,18 @@ var (
 
 // SpanHandlerBuilder builds span (Jaeger and zipkin) handlers
 type SpanHandlerBuilder interface {
-	BuildHandlers() (app.ZipkinSpansHandler, app.JaegerBatchesHandler, error)
+	BuildHandlers(cOps *CollectorOptions) (app.ZipkinSpansHandler, app.JaegerBatchesHandler, error)
```

Comment:
this doesn't look right - configuration of the builder should happen during its construction.